src/parser/jobs/blm/modules/Gauge.js

```diff
@@ -31,6 +31,7 @@ export default class Gauge extends Module {
 		'precastAction', // eslint-disable-line @xivanalysis/no-unused-dependencies
 		'suggestions',
 		'brokenLog',
+		'unableToAct',
```

Comment:
Any reason for the preference on `unableToAct`? `downtime` collates the data from both it and `invuln`, the latter of which is going to (eventually) be the one that reads downtime events

Reply:
Umbral Soul can still be used to maintain AF/UI when the boss is invulnerable since it doesn't require a target, so we need to specifically look when when you couldn't do anything at all in order to say that dropping Enochian was ok.

Reply:
Ah, of course.